src/analysis/schwartz_space.lean

```diff
@@ -403,6 +405,43 @@ begin
   rwa [pow_zero, one_mul] at this,
 end
 
+lemma le_sup_seminorm {k n k' n' : ℕ} (hk : k' ≤ k) (hn : n' ≤ n) :
+  (seminorm 𝕜 k' n' : seminorm 𝕜 𝓢(E, F)) ≤ (finset.Iic (k, n)).sup (λ n, seminorm 𝕜 n.1 n.2) :=
```

Comment:
```suggestion
lemma le_sup_seminorm {M : ℕ × ℕ} {k' n' : ℕ} (hk : k' ≤ M.1) (hn : n' ≤ M.2) :
  (seminorm 𝕜 k' n' : seminorm 𝕜 𝓢(E, F)) ≤ (finset.Iic M).sup (λ m, seminorm 𝕜 m.1 m.2) :=
```
looks more readable to me: it avoids writing a pair when what you really want is just some way to get a bound. Also, using `m` instead of `n` in the variable name can avoid some confusions. Same thing below several times.

Maybe add a docstring as the meaning of `finset.Iic M` is maybe not so obvious?

Reply:
Thanks, this is a very good point. I've added the note to all lemmas that involve `finset.Iic m`.